### DISCUSSION TITLE
rubyのコードを実行するための環境を整える

### DISCUSSION THREAD

**Discussion by kakeru-one** [sho-work/meta_programming_ruby#3] [Ideas]
## やりたいこと
- bundler/inlineでgemを使用できるようにする。
  - pry-byebugを使えるようにしたい。
- rubyのバージョンを3.3にする。
- irbではなく、pryを使えるようにする。

**Comment by kakeru-one** [comment]
gemのコードにコードジャンプできると良さそう。

**Reply by shogokaji** [reply]
code --install-extension rebornix.ruby

**Reply by kakeru-one** [reply]
今使うならこれがいいかもですね〜
https://zenn.dev/necocoa/articles/vscode-ruby-lsp

**Reply by shogokaji** [reply]
普段はこれ使ってます！
ただ、これだけだと関数の定義ジャンプができなくて、併用しているって感じです。。

**Reply by sho-work** [reply]
なるほど、それはコントリビュートチャンスですね〜！
